src/views/statusView.ts

```diff
@@ -154,7 +154,7 @@ export default class StatusView implements vscode.Disposable {
         return bar;
     }
 
-    public show(fileUri: string): void {
+    public connected(fileUri: string): void {
```

Comment:
Can you rename this (and the other similar calls) to `setConnected` so it's obvious from the function name that this performs an action.